packages/appeals-service-api/src/routes/v2/appeal-cases/_caseReference/lpa-statement-submission/index.js

```diff
@@ -0,0 +1,15 @@
+const express = require('express');
+const {
+	getLPAStatementSubmission,
+	patchLPAStatementSubmission,
+	createLPAStatementSubmission
+} = require('./controller');
+const asyncHandler = require('@pins/common/src/middleware/async-handler');
+const router = express.Router({ mergeParams: true });
+
```

Comment:
can we add auth to the endpoints please
generic token auth
and ownership check checking user belongs to lpa the case is assigned to

submissionRepo.lpaCanModifyCase